terraform/Dockerfile

```diff
@@ -29,3 +29,8 @@ RUN wget https://github.com/infracost/infracost/releases/download/v${INFRACOST_V
 
 RUN wget https://raw.githubusercontent.com/infracost/infracost/v${INFRACOST_VERSION}/scripts/ci/diff.sh -O /opt/diff.sh \
     && chmod +x /opt/diff.sh
+
+RUN echo "deb https://apt.boltops.com stable main" > /etc/apt/sources.list.d/boltops.list \
+    && curl -s https://apt.boltops.com/boltops-key.public | apt-key add - \
```

Comment:
could you run this before the first apt-get update, to avoid doing 2 apt-get updates

Reply:
@axelpavageau "could you run this before the first apt-get update, to avoid doing 2 apt-get updates" => I s not possible with the current structure of the dockerFile I need` gnupg` to be installed before  adding the mirrors of terraspace to apt.

Reply:
done in the last commit we used wget instead of curl